app/routes/team.js

```diff
@@ -27,6 +28,13 @@ router.post(
   teamController.createTeam
 );
 
+router.get(
+  '/login/:link',
```

Comment:
Is this how we said we wanted to do this? I thought we were gonna post to /team/login and provide the code in the body?